bundles/org.openhab.binding.senseenergy/src/main/java/org/openhab/binding/senseenergy/internal/handler/SenseEnergyBridgeHandler.java

```diff
@@ -120,7 +118,7 @@ && getThing().getStatusInfo().getStatusDetail() == ThingStatusDetail.COMMUNICATI
         // token is verified on each api call, called here in case no API calls are made in the alloted period
         try {
             getApi().verifyToken();
-        } catch (InterruptedException | TimeoutException | ExecutionException | SenseEnergyApiException e) {
+        } catch (SenseEnergyApiException e) {
```

Comment:
Why changed this, we prefer the catch of explicit exceptions over the generic `Exception`